mne/tests/test_filter.py

```diff
@@ -197,6 +199,24 @@ def test_iir_stability():
     assert_allclose(x_sos[100:-100], x_ba[100:-100])
 
 
+def test_iir_phase():
+    """Test IIR filter phase."""
```

Comment:
Can you add some comments explaining what you are testing here? Otherwise future developers will have a hard time figuring it out. Just the basic idea (making sure that the peak is not shifted for zero-phase, but gets delayed for the causal IIR).